testsuite/features/support/env.rb

```diff
@@ -205,6 +220,22 @@ def process_code_coverage
   skip_this_scenario
 end
 
+# Create a user for each feature
+Before do |scenario|
```

Comment:
If I'm correct, this step is also going to be executed for secondary non parallel features. You should probably differentiate the secondary and parallel secondary features to not execute for non parallel feature.

Reply:
That gonna come later. For now this refactor will indeed cause the creation of a user in all the features, but that should not be a big deal.

The next PR I have in mind, @uyuni-project/qe for everyone awareness, is to organize the secondary features in a way that we can identify which ones can run in parallel and which ones don't support it. As for now, we can only identify it through our YAML files.

I'm open to proposals, my two PoCs gonna be : 
- A cucumber tag on the top of every feature: "@parallel" or "@sequential"
- Split secondary in two subfolders: "sequential" and "parallel"